docs/ExceptionHandling.md

```diff
@@ -0,0 +1,9 @@
+# Exception handling
+
+- `on:except:do:` provides an extension to the exception handling machinery to allow handling an exception, but ignoring some subclass of it. For example:
```

Comment:
...provides an extension to the exception handling allowing to ignore some subclass of the exception being handled. For example: